Tabloid/client/src/components/PostForm.js

```diff
@@ -87,7 +87,9 @@ export const PostForm = () => {
                         onInput={(e) => setPublicationDate(e.target.value)}
                     />
                 </FormGroup>
-                <Button onClick={makePost}>Submit</Button>
+                <Button disabled={!title || !content || !chosenCategory || chosenCategory === 0} onClick={makePost}>
```

Comment:
Is the category nullable? if so, perhaps the user should be allowed to leave that one blank?

Reply:
Category is not nullable. The Post table has CategoryId as a NOT NULL Int, and #42 has category listed as non-optional.

Reply:
Gotcha! Thanks for clarifying! You're all set then.